docs/guide/coverage.md

```diff
@@ -37,6 +37,99 @@ npm i -D @vitest/coverage-istanbul
 ```
 :::
 
+## V8 provider
+
+::: info
+The description of V8 coverage below is Vitest specific and does not apply to other test runners.
+Since `v3.2.0` Vitest has used [AST based coverage remapping](https://vitest.dev/blog/vitest-3.2.html#coverage-v8-ast-aware-remapping) for V8 coverage, which produces identical coverage reports to Istanbul.
+
+This allows users to have the speed of V8 coverage with accuracy of Istanbul coverage.
+:::
+
+By default Vitest uses `'v8'` coverage provider.
+This provider requires Javascript runtime that's implemented on top of [V8 engine](https://v8.dev/), such as NodeJS, Deno or any Chromium based browsers such as Google Chrome.
+
+Coverage collection is performed during runtime by instructing V8 using [`node:inspector`](https://nodejs.org/api/inspector.html) and [Chrome DevTools Protocol](https://chromedevtools.github.io/devtools-protocol/tot/Profiler/) in browsers. User's source files can be executed as-is without any pre-instrumentation steps.
+
+- ✅ Recommended option to use
+- ✅ No pre-transpile step. Test files can be executed as-is.
+- ✅ Faster execute times than Istanbul.
+- ✅ Lower memory usagethan Istanbul.
+- ✅ Coverage report accuracy is as good as with Istanbul ([since Vitest `v3.2.0`](https://vitest.dev/blog/vitest-3.2.html#coverage-v8-ast-aware-remapping)).
+- ⚠️ In some cases can be slower than Istanbul, e.g. when loading lots of different modules. V8 does not support limiting coverage collection to specific modules.
+- ⚠️ There are some minor limitations set by V8 engine. See [`ast-v8-to-istanbl` | Limitations](https://github.com/AriPerkkio/ast-v8-to-istanbul?tab=readme-ov-file#limitations).
+- ❌ Does not work on environments that don't use V8, such as Firefox or Bun. Or on environments that don't expose V8 coverage via profiler, such as Cloudflare Workers.
+
+<div style="display: flex; align-items: stretch; padding: 2rem 0; max-width: 100%; overflow: auto">
```

Comment:
Wouldn't vertical be better?

Reply:
It would take quite much space of the page, that's why I chose vertical with scrolling. 🤔 

<img src="https://github.com/user-attachments/assets/0ccc19c8-c29b-46f3-b334-6a1a7c22a1ac" width="300" />

<img src="https://github.com/user-attachments/assets/3d7adc40-5ffc-4e51-b1d9-0f071a9fa3ea" width="300" />